src/utils/DirectoryUtils.ts

```diff
@@ -0,0 +1,20 @@
+import { readdir } from "fs";
+import { promisify } from "util";
+
+class DirectoryUtils {
+    static readDirectory = promisify(readdir);
+
+    private static require = require;
+
+    static async getFilesInDirectory(directory: string, ending: string): Promise<any[]> {
+        const directoryContents = await this.readDirectory(directory);
+
+        /* eslint-disable */
+        return directoryContents
+            .filter(file => file.endsWith(ending))
+            .map(file => this.require(`${directory}/${file}`));
+        /* eslint-enable */
+    }
```

Comment:
The linter complains: `Expected indentation of 5 tabs, found 8 spaces`. 
However it's rendering fine here and on my editor. If I try to quick fix it I still get `Mixed indentation of tabs and spaces`, which renders awfully.

Reply:
I disabled the linter for that part of the code. :p